specs/deneb/p2p-interface.md

```diff
@@ -217,34 +218,24 @@ Request Content:
 ```
 
 Response Content:
-
-```python
-class BlobSidecars(Container):
-  block_root: Root
-  List[BlobSidecar, MAX_BLOBS_PER_BLOCK]
-```
-
 ```
 (
-  List[BlobSidecars, MAX_REQUEST_BLOB_SIDECARS]
+  List[BlobSidecar, MAX_REQUEST_BLOB_SIDECARS * MAX_BLOBS_PER_BLOCK]
 )
 ```
 
-Requests blob sidecars in the slot range `[start_slot, start_slot + count)`,
-leading up to the current head block as selected by fork choice.
+Requests blob sidecars in the slot range `[start_slot, start_slot + count)`, leading up to the current head block as selected by fork choice.
 
-The response is unsigned, i.e. `BlobSidecarsByRange`, as the signature of the beacon block proposer
-may not be available beyond the initial distribution via gossip.
+The response is unsigned, i.e. `BlobSidecarsByRange`, as the signature of the beacon block proposer may not be available beyond the initial distribution via gossip.
 
-Before consuming the next response chunk, the response reader SHOULD verify the blobs sidecar is well-formatted and
-correct w.r.t. the expected KZG commitments through `validate_blobs_sidecar`.
+Before consuming the next response chunk, the response reader SHOULD verify the blobs sidecar is well-formatted and correct w.r.t. the expected KZG commitments through `validate_blobs_sidecar`.
```

Comment:
`validate_blobs_sidecar` function no longer exists

It's also unclear if you should do this with each blob sidecar or with each batched set per block. Maybe it doesn't matter as both can work with the `validate_blobs` function

```suggestion
Before consuming the next response chunk, the response reader SHOULD verify the blob sidecar is well-formatted and correct w.r.t. the expected KZG commitments through `validate_blobs`.
```